dev/AppNotifications/AppNotifications.idl

```diff
@@ -123,6 +123,10 @@ namespace Microsoft.Windows.AppNotifications
         // For Unpackaged apps, the caller process will be registered as the COM server. And assets like displayname and icon will be gleaned from Shell and registered as well.
         void Register();
 
+        // For Unpackaged apps only, the caller process will be registered as the COM server.
```

Comment:
This statement is untrue.

Reply:
Packaged apps are also registered as a COM server for AppNotifications

Reply:
The only difference is that we define the COM server in a manifest for packaged apps. For unpackaged, we write to registry.

After that we call CoRegister on both.